components/browser/errorpages/src/main/res/values/strings.xml

```diff
@@ -129,7 +131,7 @@ be temporary, and you can try again later.</li>
 <p>The browser has stopped trying to retrieve the requested item. The site is redirecting the request in a way that will never complete.</p>
 <ul>
 <li>Have you disabled or blocked cookies required by this site?</li>
-<li><em>NOTE</em>: If accepting the site\'s cookies does not resolve the problem, it is likely a server configuration issue and not your computer.</li>
+<li><em>NOTE</em>: If accepting the site\’s cookies does not resolve the problem, it is likely a server configuration issue and not your computer.</li>
 </ul>
      ]]></string>
 
```

Comment:
in the [`Offline Mode` string](https://github.com/mozilla-mobile/android-components/blob/b30888/components/browser/errorpages/src/main/res/values/strings.xml#L143) can we change `&quot;Try Again&quot;` to `“Try Again”`?